### PR TITLE
fix and improve(admin): notifications

### PR DESCRIPTION
**Pull request by thimy**
Changed notification style to toast 🍞
Fixed the notifications to be displayed on non-wizard 🎩 layout. (https://github.com/opendatateam/udata/issues/928)

image

### CODE REVIEW

**Comment by abulte** [comment]
@thimy does Vue 1 supports mixin? The `validate()` method seems very repetitive, maybe it can be mutualized in a mixin. My 2 cents, I obviously don't know the code and/or best practices for the project at all at this point ;-)

File: js/components/dataset/resource/form.vue [hunk 1]
```diff
@@ -101,6 +101,7 @@ export default {
             type: Object,
             default() {return new Resource()}
         },
+        notifications: true
```
Comment (noirbizarre) [inline js/components/dataset/resource/form.vue]: This is the only one with positive variable (ie. `notifications` vs `hideNotifications`).
We should stay consistent and chose between one of the naming (personnaly I prefer `notifications` over `hideNotifications`)

Comment (thimy) [inline js/components/dataset/resource/form.vue]: Oops, forgot this one!

**Comment by noirbizarre** [comment]
@abulte Yes, Vue 1.x support mixins.
I think the validation pattern is not the focus of this PR and should be handled in a separate PR (see the #526 EPIC, seems to fit in "jquery-validation (admin form validation)").
About conventions, we have some work to do like this #753 :wink:

**Comment by thimy** [comment]
@abulte I'm not sure, not exactly acquainted to Vue yet, I'll look into this. 👍

**Comment by thimy** [comment]
Notifications were indeed displayed in modals as well, fixed along with variable name.

**Pull request merged**